examples/SimpleModule/SimpleModule.csproj

```diff
@@ -21,4 +21,11 @@
       </None>
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageReference Include="SonarAnalyzer.CSharp" Version="9.5.0.73987">
```

Comment:
What is this added to all the projects?

Reply:
This is so SonarLint can analyze all classes in the IDE so you can fix code smells, potential bugs etc. in your IDE instead of having to run SonarCloud every single time. Sort of an extra code quality addon.